frontend/public/components/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -24,14 +24,14 @@ const overviewQueries = {
   [OverviewQuery.CPU_UTILIZATION]: '((sum(node:node_cpu_utilisation:avg1m) / count(node:node_cpu_utilisation:avg1m)) * 100)[60m:5m]',
   [OverviewQuery.STORAGE_UTILIZATION]: '(sum(node_filesystem_size_bytes) - sum(node_filesystem_free_bytes))[60m:5m]',
   [OverviewQuery.STORAGE_TOTAL]: 'sum(node_filesystem_size_bytes)',
-  [OverviewQuery.PODS_BY_CPU]: 'sort_desc(pod_name:container_cpu_usage:sum)',
+  [OverviewQuery.PODS_BY_CPU]: 'sort_desc(pod_name:container_cpu_usage:sum)*1000^3',
   [OverviewQuery.PODS_BY_MEMORY]: 'sort_desc(pod_name:container_memory_usage_bytes:sum)',
-  [OverviewQuery.PODS_BY_STORAGE]: 'sort_desc(avg by (pod_name)(irate(container_fs_io_time_seconds_total{container_name="POD", pod_name!=""}[1m])))',
+  [OverviewQuery.PODS_BY_STORAGE]: 'sort_desc(avg by (pod_name)(irate(container_fs_io_time_seconds_total{container_name="POD", pod_name!=""}[1m])))*1000^3',
```

Comment:
I assume this is to show Gb values. This should be done at presentation layer, not in the query. Queries and query results should always be in base units.

Reply:
it shows average I/O time, not Gb values. The reasoning is same as above.